src/models/lemp9/README.md

```diff
@@ -1,50 +0,0 @@
-# Lemur Pro (lemp9)
```

Comment:
Was deleting this file intentional?

Reply:
No.

Reply:
yes, they shouldn't have been in this branch. They were causing conflicts with the master branch.